tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+
+### Multi-Temporal Data in Earth Engine
+
+Time series data in Earth Engine are represented as a series of images called 'Image Collections'. As a result of the complicating factors in remote sensing discussed earlier, analyzing time series in Earth Engine is unlike time series modeling in traditional methods. From a programming sense, we will join data together to define temporal relationships between collection items and build functions to reduce this time. 
+
+First, some very basic mathematical notation for time series. A time series is an array of the value being measured, sorted chronologically: { $\textbf{p}_{t}  = t_{0} + t_{1}... t_{N}$ }, where each *t* is the given value in the series.
+
+
+### Data Preparation and Preprocessing
+The first step in analysis of time series data is to import data of interest and plot the data around our region of interest, a deciduous forest near Blacksburg, VA.
+
+We begin by loading in the Landsat 8 Collection and provide a point at the region of interest. Additionally, we will create a time field. 
+
+```javascript
+var landsat_8_sr = ee.ImageCollection("LANDSAT/LC08/C01/T1_SR")
+var roi = ee.Geometry.Point([-80.49882800809357, 37.2544486695189]);
+// This field contains UNIX time in milliseconds.
+var timeField = 'system:time_start';
+```
+
+The function `maskL8sr` is a cloud masking function that uses the Quality Assurance attribute of Landsat 8 to mask out any pixels that that are obscured by cloud. Note that this function is Landsat 8 specific, using other platforms will require a different setup. 
+
+```javascript
+// Function to cloud mask from the pixel_qa band of Landsat 8 SR data.
+function maskL8sr(image) {
+  // Bits 3 and 5 are cloud shadow and cloud, respectively.
+  var cloudShadowBitMask = 1 << 3;
+  var cloudsBitMask = 1 << 5;
+  // Get the pixel QA band.
+  var qa = image.select('pixel_qa');
+  // Both flags should be set to zero, indicating clear conditions.
+  var mask = qa.bitwiseAnd(cloudShadowBitMask).eq(0)
+  .and(qa.bitwiseAnd(cloudsBitMask).eq(0));
+  // Return the masked image, scaled to reflectance, without the QA bands.
+  return image.updateMask(mask).divide(10000)
+  .select('B[0-9]*')
+  .copyProperties(image, [timeField]);
+}
+```
+
+We can use one of the indices that we built in an earlier lab to measure vegetation health. Normalized Difference Vegetation Index (NDVI) is a well-known metric for quantifying vegetation health - for this region of interest, we expect there to be a strong seasonality, and perhaps a gradual linear trend over time. In the code block below, we create a function called `addVariables` that extracts the date of each image, calculates NDVI and adds it to an array. We can then use `.map()` to apply the two functions we built to build a time series model of our data. 
```

Comment:
- Recommend to shorten first sentence to:

> We'll use the Normalized Difference Vegetation Index (NDVI), a well-known metric for measuring quantifying vegetation productivity, as our time series signal.

- In the last sentence, replace "built" with "defined"

Reply:
Updated both sentences